hack/dci-merge.sh

```diff
@@ -71,17 +71,17 @@ for PR in $PRS; do
         # extract TestBos2 commands
         if grep -qE "^\s*TestBos2:\s*" <<< "$DESC"; then
             # shellcheck disable=SC2001,SC2086
-            CMD+=("$(sed -e 's/^\s*TestBos2:\s*//' <<< $DESC)")
```

Comment:
I think it is easier to read with one sed command like that:

`CMD+=("$(sed -ne 's/^\s*TestBos2:\s*//p' <<< $DESC)")`

Reply:
ack, updated